tests/manager/api/test_app.py

```diff
@@ -0,0 +1,20 @@
+from unittest.mock import patch
+
+from palace.manager.api.app import initialize_application
+from palace.manager.util.http import HTTP
+from tests.fixtures.database import DatabaseTransactionFixture
+from tests.fixtures.services import ServicesFixture
+
+
+def test_initialize_application_http(
+    db: DatabaseTransactionFixture, services_fixture_wired: ServicesFixture
+):
+    # Use the db transaction fixture so that we don't use the production settings by mistake
+    with patch.object(
+        HTTP, "set_quick_failure_settings"
+    ) as mock_set_quick_failure_settings:
+        # Initialize the app, which will set the HTTP configuration
+        initialize_application()
+
+    # Make sure that the HTTP configuration was set
+    mock_set_quick_failure_settings.assert_called_once()
```

Comment:
This test was updated to use a mock. Previously the call it was making to `initialize_application` was changing the global state for HTTP module, which caused other tests to fail. Using a mock here does the same test, but prevents this side effect.